### PR TITLE
Add server command help for the checkpoint action.

### PR DESCRIPTION
**Pull request by tjwatson**

### CODE REVIEW

**Comment by tjwatson** [comment]
#build

File: dev/com.ibm.ws.kernel.boot/resources/com/ibm/ws/kernel/boot/resources/LauncherOptions.nlsprops [hunk 1]
```diff
@@ -56,6 +56,16 @@ javaAgent.desc=\
 \tthe jar used to launch the runtime.
 #------------------------------\n at 72 chars -- leading tab-----------\n\#
 # Actions: create, stop, version, package, dump
+action-key.checkpoint=\
+\ \ \ \ --checkpoint
+action-desc.checkpoint=\
+\tCheckpoint the named server at the specified checkpoint phase. The  \n\
```
Comment (dmuelle) [inline dev/com.ibm.ws.kernel.boot/resources/com/ibm/ws/kernel/boot/resources/LauncherOptions.nlsprops]:
```suggestion
\tCheckpoint the named server at the specified checkpoint phase. Use  \n\
```

File: dev/com.ibm.ws.kernel.boot/resources/com/ibm/ws/kernel/boot/resources/LauncherOptions.nlsprops [hunk 2]
```diff
@@ -56,6 +56,16 @@ javaAgent.desc=\
...
+\t--at option can be used to specify the checkpoint phase. When the   \n\
```
Comment (dmuelle) [inline dev/com.ibm.ws.kernel.boot/resources/com/ibm/ws/kernel/boot/resources/LauncherOptions.nlsprops]:
```suggestion
\tthe --at option to specify the checkpoint phase. When the   \n\
```

File: dev/com.ibm.ws.kernel.boot/resources/com/ibm/ws/kernel/boot/resources/LauncherOptions.nlsprops [hunk 3]
```diff
@@ -56,6 +56,16 @@ javaAgent.desc=\
...
+\tcheckpoint completes, the server stops and its current state is     \n\
+\twritten to disk. The checkpoint is used to provide InstantOn        \n\
```
Comment (dmuelle) [inline dev/com.ibm.ws.kernel.boot/resources/com/ibm/ws/kernel/boot/resources/LauncherOptions.nlsprops]:
```suggestion
\twritten to disk. This checkpoint is provides InstantOn        \n\
```

File: dev/com.ibm.ws.kernel.boot/resources/com/ibm/ws/kernel/boot/resources/LauncherOptions.nlsprops [hunk 4]
```diff
@@ -56,6 +56,16 @@ javaAgent.desc=\
...
+\tbehavior when launching the server. When the checkpoint completes,  \n\
```
Comment (dmuelle) [inline dev/com.ibm.ws.kernel.boot/resources/com/ibm/ws/kernel/boot/resources/LauncherOptions.nlsprops]:
```suggestion
\tbehavior when the server launches. When the checkpoint completes,  \n\
```

File: dev/com.ibm.ws.kernel.boot/resources/com/ibm/ws/kernel/boot/resources/LauncherOptions.nlsprops [hunk 5]
```diff
@@ -56,6 +56,16 @@ javaAgent.desc=\
...
+\tthe server can be launched with InstantOn using the run or start    \n\
```
Comment (dmuelle) [inline dev/com.ibm.ws.kernel.boot/resources/com/ibm/ws/kernel/boot/resources/LauncherOptions.nlsprops]:
```suggestion
\tyou can launch the server with InstantOn by using the run or start    \n\
```

**Review by dmuelle — COMMENTED** [review]
ID review

**Comment by tjwatson** [comment]
#build

**Review by dmuelle — APPROVED** [review]
LGTM, thanks

**Review by anjumfatima90 — APPROVED** [review]

**Comment by tjwatson** [comment]
#libby

**Pull request merged**